spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseSparkAction.java

```diff
@@ -296,6 +315,11 @@ public long totalFilesCount() {
     }
   }
 
+  protected Dataset<Row> filterAllManifests(Dataset<Row> allManifestDF, Set<Long> snapshots) {
```

Comment:
Nit: `snapshots` would be more clear if it were named `snapshotIds`.

Reply:
Refactored snapshots -> snapshotIds in these files